scripts/superdesk-search/search.js

```diff
@@ -2659,7 +2659,14 @@
                          */
                         function getQuery() {
                             var metas = [];
+
                             angular.forEach(scope.meta, function(val, key) {
+                                //checkbox boolean values.
+                                if (typeof(val) === 'boolean') {
+                                    val = Number(val);
```

Comment:
Yeah, in PA images API the Boolean types needed are 0/1.

I don't mind micro level comments :smile: